torchx/schedulers/aws_batch_scheduler.py

```diff
@@ -588,13 +641,35 @@ def log_iter(
     def list(self) -> List[ListAppResponse]:
         # TODO: get queue name input instead of iterating over all queues?
         all_apps = []
-        for resp in self._client.get_paginator("describe_job_queues").paginate():
-            queue_names = [queue["jobQueueName"] for queue in resp["jobQueues"]]
-            for qn in queue_names:
-                all_apps.extend(self._list_by_queue(qn))
+        regions = _local_sessions_available_regions()
+        for region in regions:
```

Comment:
have you measured how long this takes? last I tried this, it drastically worsens latency for m5 since we have a ton of job queues across regions. (by drastic I mean ~40seconds for the list API)

Reply:
Ill double-check for M5. However, did this even work across regions originally? A more apt comparison might be specifying the region in the .torchxconfig vs just using the other methods which I think I originally did.